docs/how_xtrabackup_works.md

```diff
@@ -12,6 +12,8 @@ steps. It applies committed transaction log entries to the data files, and it
 performs an undo operation on any transactions that modified data but did not
 commit.
 
+[Percona XtraBackup 8.0.30-23](release-notes/8.0/8.0.30-23.0.md) adds `new --register-redo-log-consumer` parameter. This parameter allows registering Percona XtraBackup as a redo log consumer at the start of the backup. The `new --register-redo-log-consumer` parameter keeps informing the server to up which position of redo log we have already consumed and whether it is safe to purge the data. If Percona XtraBackup is lagging, the server does not remove a redo log that Percona XtraBackup (the consumer) has not yet copied. If the server needs to make room in the redo logs for new data when Percona XtraBackup is lagging, the server blocks the writes until Percona XtraBackup advances, and there is room in redo logs. Thus, the users can control if the backup is successful. The `new --register-redo-log-consumer` parameter is disabled by default. 
```

Comment:
new should not be highlighted in --register...
informing the server to up which position of redo log - not clear. Maybe "The consumer continually informs the server of their progress by manually advancing the LSN, and the server determines whether it is safe to purge the log."

Reply:
Fixed